src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala

```diff
@@ -708,9 +708,9 @@ abstract class ClassfileParser(reader: ReusableInstance[ReusableDataReader]) {
           while (sig.charAt(index) == '.') {
             accept('.')
             val name = newTypeName(subName(c => c == ';' || c == '<' || c == '.'))
-            val clazz = tpe.member(name)
+            val sym = if (tpe.typeSymbol == clazz) instanceScope.lookup(name) else tpe.member(name)
```

Comment:
`clazz` is now the `clazz` being completed.

Reply:
Suggest `member` because `sym` shadows a parameter.